src/backends/x11.rs

```diff
@@ -163,6 +166,7 @@ enum Buffer {
     Wire(Vec<u32>),
```

Comment:
Would this print the entire pixel buffer?

Reply:
Yeah, I guess so. Might make sense to make a small wrapper to avoid that.